DEVELOPMENTLOG.md

```diff
@@ -1,3 +1,39 @@
+# OpenLoco version 23.03+ (???)
+
+## Refactor arg structs for game commands (#1893, #1916)
+
+As part of our ongoing efforts to reimplement all game commands in C++, we are refactoring the game command's argument structs to be higher-level. Currently, most are still passing registers into vanilla functions. We have devised a wrapper structure to make our C++ code more readable in the mean time, while at the same time preparing for command serialization for future multiplayer.
+This month, @duncanspumpkin and @reinaldorauch tackled a bunch of these refactors, even if not all of them have been merged yet.
+
+## Implement two more vehicle game commands (#1923, #1924)
+
+Related to the previous section, @AaronVanGeffen implemented two more vehicle game commands: the vehicle 'reverse' command, and the vehicle 'pass signal' command.
+
+## Improved diagnostics and tests (#1894, #1904)
+
+@ZehMatt reworked out console logging interfaces into a new Diagnostics module. This allows log messages to easily be saved to file for reference, and for easier extension and reuse in the future. In addition, more tests for our custom string functions were added. Hopefully, this will prevent regressions from popping up when we switch to Unicode for internal strings in the future.
```

Comment:
reworked out sounds odd. Should that be reworked our?

Reply:
```suggestion
@ZehMatt reworked our console logging interfaces into a new Diagnostics module. This allows log messages to easily be saved to file for reference, and for easier extension and reuse in the future. In addition, more tests for our custom string functions were added. Hopefully, this will prevent regressions from popping up when we switch to Unicode for internal strings in the future.
```